### PR TITLE
Don't use custom Bundler path

### PR DESCRIPTION
**Pull request by fkorotkov**

### CODE REVIEW

**Comment by AlexWayfer** [comment]
It was made because of `macOS` in #172

There is Brew and a bit another installation. I wanted to unify it.

Also I had similar in some repos, but without macOS, for example: https://github.com/r18n/sinatra-r18n/blob/master/.cirrus.yml

Are you sure about these changes?

**Comment by AlexWayfer** [comment]
Especially in this PR I see:

> Cache miss for bundle-65e14e12f77284fd0ba506bc37e3792cf1d2a2661895ec59e97182055fb06ee4! No script to populate with.

https://cirrus-ci.com/task/6467993317146624?command=bundle#L54

> Skipping change detection for bundle cache!

https://cirrus-ci.com/task/6467993317146624?command=Upload%20%27bundle%27%20cache#L0

WTF? 😅

**Comment by fkorotkov** [comment]
This is not WTF. This is coming from macOS where is can't write to `/usr/local/bundle` https://cirrus-ci.com/task/6467993317146624?command=bundle#L50

**Comment by AlexWayfer** [comment]
> This is not WTF. This is coming from macOS where is can't write to `/usr/local/bundle` https://cirrus-ci.com/task/6467993317146624?command=bundle#L50

OK, thank you, so… maybe we should use relative custom path for which we have permissions to write and read?

File: .cirrus.yaml [hunk 1]
```diff
@@ -1,5 +1,22 @@
+# The default 2 CPUs / 4G memory is not required.
+container:
+  cpu: 1
+  memory: 2G
+
 unix_bundle_cache: &unix_bundle_cache
   bundle_cache:
+    folder: /usr/local/bundle
```
Comment (AlexWayfer) [inline .cirrus.yaml:8]: But why do you suggest to use absolute path for Linux if relative has no problems in other projects? Like, again, [`sinatra-r18n`](https://github.com/r18n/sinatra-r18n): https://cirrus-ci.com/task/6211249500323840

Comment (fkorotkov) [reply .cirrus.yaml:8]: Yeah, also not 100% sure about that. Will close PR then

Comment (AlexWayfer) [reply .cirrus.yaml:8]: > Yeah, also not 100% sure about that. Will close PR then

As I wrote, I'm thinking the issue in the CI itself, something has been broken there with cache. And I'm ready to try, to help, to investigate — just write me anywhere. I want to help not to me, but to us: make this awesome CI better and more stable. ❤️

**Pull request closed without merging**